### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: lib/AmazonECS.class.php [hunk 1]
```diff
@@ -568,4 +548,21 @@ public function requestDelay($enable = null)
...
+   */
+  public function getCountry()
+  {
+      return $this->responseConfig['country'];
+  }
+  
+  /**
+   * @return array
+   */
+  public static function getSupportedLocations()
```
Comment (Exeu) [inline lib/AmazonECS.class.php:563]: i think we dont need this here.

File: lib/AmazonECS.class.php [hunk 2]
```diff
@@ -568,4 +548,21 @@ public function requestDelay($enable = null)
 
     return $this->requestConfig['requestDelay'];
   }
-}
\ No newline at end of file
+  
+  /**
+   * @return string
+   */
+  public function getCountry()
```
Comment (Exeu) [inline lib/AmazonECS.class.php:555]: We allready have a getter for this.

File: lib/AmazonECS.class.php [hunk 3]
```diff
@@ -48,21 +57,7 @@ class AmazonECS
...
-   */
-  protected $webserviceWsdl = 'http://webservices.amazon.com/AWSECommerceService/AWSECommerceService.wsdl';
-
-  /**
-   * The SOAP Endpoint
-   *
-   * @var string
-   */
-  protected $webserviceEndpoint = 'https://webservices.amazon.%%COUNTRY%%/onca/soap?Service=AWSECommerceService';
+  protected static $supportedLocations = array('de', 'com', 'co.uk', 'ca', 'fr', 'co.jp', 'it', 'cn', 'es');
```
Comment (Exeu) [inline lib/AmazonECS.class.php:60]: What is the reason for declaring this as static?

File: lib/AmazonECS.class.php [hunk 4]
```diff
@@ -22,6 +22,15 @@ class AmazonECS
 {
   const RETURN_TYPE_ARRAY  = 1;
   const RETURN_TYPE_OBJECT = 2;
+  /**
+   * The WSDL File
+   */
+  const WS_WSDL = 'http://webservices.amazon.com/AWSECommerceService/AWSECommerceService.wsdl';
```
Comment (Exeu) [inline lib/AmazonECS.class.php:28]: Why using a constant for this?

File: lib/AmazonECS.class.php [hunk 5]
```diff
@@ -22,6 +22,15 @@ class AmazonECS
...
+
+  /**
+   * The SOAP Endpoint
+   */
+  const WS_ENDPOINT = 'https://webservices.amazon.%%COUNTRY%%/onca/soap?Service=AWSECommerceService';
```
Comment (Exeu) [inline lib/AmazonECS.class.php:33]: Why using a constant for this?

**Comment by Exeu** [comment]
I'ive commented a few lines of code. Please read it

File: lib/AmazonECS.class.php (hunk 3 above)
Comment (elvisciotti) [inline lib/AmazonECS.class.php:60]: The reasons is reading them without constructing the class

File: lib/AmazonECS.class.php (hunk 4 above)
Comment (elvisciotti) [inline lib/AmazonECS.class.php:28]: because the WSDL url basically never changes

**Comment by elvisciotti** [comment]
I applied those changes for simplicity of the logic of one of my project. I understand the generic needs are different, so feel free to close this

**Pull request closed without merging**